src/components/Lobby/Lobby.tsx

```diff
@@ -42,23 +46,25 @@ function Lobby({players}: Props) {
         navigator.clipboard.writeText(joinURL);
     };
 
-
-    const startGame = () => {
-        navigate(`/game/play/${gameid}`, {state: sessionGame});
-        //instead, pass new object with up to date players and game questions
-        //and other needed details until websockets are set up
-        
-        //make started game post request, update respective game state property
-    }
+    const startGame = async () => {
+        try {
+            await patchGame(gameid);
+            navigate(`/game/play/${gameid}`, {state: sessionGame});
```

Comment:
I'm thinking that this is one of the places that can be refactored once a WebSocket connection is opened!